client/scss/utilities/_variables.scss

```diff
@@ -254,13 +254,17 @@ $colors: (
   'cream': (#f0ede3, 'primary'),
   'black': (#1d1d1d, 'primary'),
   'dark-black': (#010101, 'primary'),
-  'turquoise': (#489199, 'secondary'),
+  'turquoise-background': (#5cb8bf, 'secondary'),
+  'turquoise-foreground': (#367378, 'secondary'),
   'teal': (#006272, 'secondary'),
   'petrol': (#00424d, 'secondary'),
-  'purple': (#8c4799, 'secondary'),
+  'purple-background': (#b67bc2, 'secondary'),
+  'purple-foreground': (#895791, 'secondary'),
   'indigo': (#46284d, 'secondary'),
-  'red': (#ff4355, 'secondary'),
-  'orange': (#e87500, 'secondary'),
+  'red-background': (#ff4355, 'secondary'),
+  'red-foreground': (#c72e3d, 'secondary'),
+  'orange-background': (#e87500, 'secondary'),
+  'orange-foreground': (#ad4e00, 'secondary'),
```

Comment:
I think that at the expense of being occasionally semantically ambiguous, this naming is probably better than e.g. red-lighter/red-darker because the intent is clearer (alarm bells should ring if we see `background: color('red-foreground')`. Reckons?